packages/wrangler/src/publish.ts

```diff
@@ -222,7 +222,9 @@ export default async function publish(props: Props): Promise<void> {
 
 	if (!(props.compatibilityDate || config.compatibility_date)) {
 		const compatibilityDateStr = `${new Date().getFullYear()}-${(
-			new Date().getMonth() + ""
+			new Date().getMonth() +
+			1 +
+			""
```

Comment:
Out of curiosity, any idea why these `""` are here?
Perhaps they were previously there when the string was being built via concatenation? And now we are using string interpolation they are not needed, right?

Reply:
It seems to have been a hack to coerce the type into a string for the following `.padStart(2, "0")` which could do this `String(new Date().getMonth() + 1)` instead which is also far more explicit of what/why its being done

Reply:
> It seems to have been a hack to coerce the type into a string for the following `.padStart(2, "0")` which could do this `String(new Date().getMonth() + 1)` instead which is also far more explicit of what/why its being done

This actually seems to change the output just slightly enough to effect the tests.